scripts/create-plugin-list.js

```diff
@@ -0,0 +1,10 @@
+import * as corePlugins from '../src/plugins'
+import fs from 'fs'
+import path from 'path'
+
+const corePluginList = Object.keys(corePlugins)
+
+fs.writeFileSync(
+  path.join(__dirname, '..', 'src', 'corePluginList.js'),
+  `export default ${JSON.stringify(corePluginList)}`
+)
```

Comment:
Q: Since this logic is only used by `resolveConfig.js` can we not generate the file and directly move this to `resolveConfig.js`?

Reply:
This would make it so we would have to commit a generated file to the repo. It would also have to conform to the linter. Is this desired behavior?

Reply:
I meant something more similar to the older logic.

Reply:
I'm confused as to what this means. We cannot do this in `resolveConfig.js` because it would mean a package consumer would have to import all the plugins and thus import CSS from the `preflight` plugin. See linked issue for why this is a problem.

Reply:
Since exports are transpiled to commonjs, there may exist some trick to get a list of all of them. Just suppositions though.